app/views/sessions/prompt.html.erb

```diff
@@ -0,0 +1,37 @@
+<% @title = t('multifactor_authentication') %>
```

Comment:
<img width="1109" alt="Screen Shot 2022-09-19 at 11 52 58 AM" src="https://user-images.githubusercontent.com/42748004/191059951-46d0fb1b-ca0a-4a5a-bfba-7a5c60684871.png">

It took me a while to figure out what to do on this page. This is because there are multiple buttons with the same label "Sign in" and took me a few seconds to figure out what each of them means. Could we have subheadings on each of sections like "Security device" and "One time passcode", and have maybe have differing button names? For aesthetic purposes, would also be great if the two sections were side by side rather than in a stack (nit).
<details>
<summary>PYPI and NPM multi-factor auth signin pages </summary>
<img width="1246" alt="Screen Shot 2022-09-19 at 12 01 41 PM" src="https://user-images.githubusercontent.com/42748004/191061687-b8f176e3-6663-41fe-b807-49b16085fd1a.png">

<img width="659" alt="Screen Shot 2022-09-19 at 12 04 55 PM" src="https://user-images.githubusercontent.com/42748004/191062290-ea6cc3e6-e930-4096-930d-9bc5e0c22107.png">

<img width="673" alt="Screen Shot 2022-09-19 at 12 05 23 PM" src="https://user-images.githubusercontent.com/42748004/191062372-dd73d000-709a-46c2-9d58-894ffbc8b502.png">
</details>

Reply:
I've updated this page to be side by side and changed the buttons to be different <img width="1002" alt="image" src="https://user-images.githubusercontent.com/8496209/191893380-85b76e33-692b-4253-b537-4fadbcbaa916.png">